website/static/js/fangorn.js

```diff
@@ -115,10 +115,14 @@ function findByTempID(parent, tmpID) {
 
 function cancelUploads (row) {
     var tb = this;
-    var filesArr = tb.dropzone.getQueuedFiles();
+    var filesArr = tb.dropzone.getActiveFiles();
+    var rejectedFilesArr = tb.dropzone.getRejectedFiles();
     for (var i = 0; i < filesArr.length; i++) {
         var j = filesArr[i];
         if(!row){
+            if (j.status === 'uploading') {
```

Comment:
@sloria Right. At first I wanted to create a status setter function in dropzone.js, but that involved the overhead of overriding dropzone. 
I couldn't think of a third solution